sprintly

```diff
@@ -587,9 +587,12 @@ class SprintlyTool:
 
         # ensure the current directory is a git repository
         directory = os.getcwd()
+        git_directory = hooks_directory = os.path.join(directory, '.git')
```

Comment:
Instead of doing multiple assignments (eg `git_directory = hooks_directory = ...`, just use `git_directory` as the base on line 591 instead of `directory`.

Otherwise, this looks solid.